components/imageInput.tsx

```diff
@@ -0,0 +1,31 @@
+import type React from 'react'
+import type { InputHTMLAttributes } from 'react'
+// import type { RefCallBack } from 'react-hook-form'
+
+export type Props = {
```

Comment:
This Props is not exported.
Then remove `export`
Just `type Props`

Reply:
I changed using in imagePreview.tsx and uploadFormHook.tsx.